concepticondata/conceptlists/Wang-2021-90.tsv

```diff
@@ -0,0 +1,91 @@
+ID	NUMBER	CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH	word_id	Chinese	gram_class	word type_2 subtypes	word type_5 subtypes	ISC_Brain_ObjNObj	ISC_Brain_ObjNObj_SE	ISC_Brain_wordNeurosynth	ISC_Brain_wordNeurosynth_SE	ISC_Brain_ObjEmoNObjNENObj	ISC_Brain_ObjEmoNObjNENObj_SE	ISC_Brain_ObjNObj_leave2_top300	ISC_Brain_leave2_top300_SE	ISC_behavior	ISC_behavior_subjectSE	ISC_behavior_wordSE	Mean_Language_Sensory	Mean_Language	Mean_Sensory	Mean_Navigation	Mean_Manipulation	Mean_Stress	Mean_Arousal	Mean_Valence	SD_Language	SD_Sensory	SD_Navigation	SD_Manipulation	SD_Stress	SD_Arousal	SD_Valence	MeanRatings_PC1	MeanRatings_PC2	SDRatings_PC1	SDRatings_PC2	Number_of_Strokes	Subjective_Familiarity	log_word_frequency	log_Char1Freq	log_AVG_Char_Freq	log_Char1SRFreq	log_AVG_SRFreq	Mean_LanguageSensory_cov_MeanArousalValence	behav_ISC_cov_MeanArousalValence	ObjNObj_overall_activation	wordNeurosynth_overall_activation	ObjEmoNObjNENObj_overall_activation
+Wang-2021-90-1	1	3951	PANDA	panda	word01	熊猫	n	1	1	0.03157831136	0.005725277125	0.01491398643	0.002724606171	0.03642705787	0.005148065643	0.03223141099	0.008705191957	0.4754556799	0.03773073205	0.09710623807	0.9284821549	6.461538462	5.925925926	3.285714286	2.12	2.444444444	4.041666667	5.074074074	0.7060180865	1.268734255	1.511857892	1.563116545	1.717183144	1.545798419	0.8286191445	0.82838	0.10773	-1.09553	0.82631	25	6.846153846	1.380211242	1.549960354	1.511511713	3.789805563	3.610682766	1.17654	1.80101	28.0634474	18.13208197	40.32718825
+Wang-2021-90-2	2	1490	RAT	rat	word02	老鼠	n	1	1	0.04242091167	0.007698848148	0.01996472156	0.004013054544	0.04090807241	0.006391648553	0.04066811153	0.01042518742	0.494632664	0.03387913939	0.09997211382	0.9137594944	6.461538462	5.888888889	2.285714286	4.32	4.703703704	4.791666667	2.851851852	0.7605665904	1.310705428	1.242969648	2.267891826	1.977076607	1.413572922	0.9885383034	1.18316	2.22168	-1.13114	1.05661	19	6.730769231	1.041392685	3.124912993	2.845125648	3.430435796	3.140435869	1.4471	1.98833	27.07729526	10.1099065	39.88220067
```

Comment:
This word means both "mouse" and "rat" in Chinese, it should despite the English gloss be mapped to "muroid (mouse or rat)".